lincheck/src/main/java/org/jetbrains/kotlinx/lincheck/execution/ActorGenerator.java

```diff
@@ -53,6 +53,10 @@ public Actor generate() {
             .map(ParameterGenerator::generate).collect(Collectors.toList()), handledExceptions);
     }
 
+    public void reset() {
+        parameterGenerators.forEach(ParameterGenerator::reset);
```

Comment:
Are unique parameter generators with the same name shared between all actors?

Reply:
That's correct

Reply:
Do you create new ones for each iteration?